frontend/src/views/settings/Global.vue

```diff
@@ -92,6 +92,91 @@
               id="branding-files"
             />
           </p>
+
+          <h3>{{ $t("settings.tusUploads") }}</h3>
+
+          <i18n path="settings.tusUploadsHelp" tag="p" class="small">
+            <a class="link" target="_blank" href="https://tus.io/"
+              >$t("settings.documentation")</a
+            >
+          </i18n>
+
+          <p>
+            <input
+              type="checkbox"
+              v-model="settings.tus.enabled"
+              id="tus-enabled"
+            />
+            {{ $t("settings.tusUploadsEnabled") }}
+          </p>
+
+          <p v-if="settings.tus.enabled">
```

Comment:
will these if statements cause Vue to hide the whole setting if its not enabled? If so that might cause a large content shift when pressing the button & lead to a worse UX. We could create one wrapper div that has an optional class that marks all other settings as disabled (for styling) & just define the disabled property on each of the inputs to reduce any layout shifting but still convey that they are no longer active since Tus is disabled.